src/browser/provider/index.js

```diff
@@ -95,11 +95,9 @@ export default class BrowserProvider {
     }
 
     async _resizeLocalBrowserWindow (browserId, width, height, currentWidth, currentHeight) {
-        if (this.resizeCorrections[browserId]) {
+        if (await browserTools.isMaximized(this.windowDescriptors[browserId])) {
```

Comment:
The error is raised here if browser was opened initially in `not-maximized` mode, then was maximized and the `resizeWindow` action called.